### ISSUE TITLE
Deleted local favorites by mistake

### ISSUE THREAD

**Issue by iliyasviel1** [Dar9586/NClientV2#619]
I accidentally emptied the app's data，then I found that the  local favorites that had been accumulated for a year had also been completely emptied，I would like to ask if there is a way to recover it

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Version used:
* Operating System version:

* NHentai link (if relevant):

**Comment by flleeppyy** [comment]
Nope. Only way to recover requires root, and even if you were rooted, chances sre that deleted data is already overwritten with no chances to recovering it
